### PR TITLE
Remove not found pages on security

### PR DESCRIPTION
**Pull request by spk**
Hi, this just remove not found pages or redirect to parking domains, cheers

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ Add cert paths to your gemspec
 
 -------
 
-### OpenPGP signing is [not recommended due to lack of support](http://www.rubygems-openpgp-ca.org/blog/nobody-cares-about-signed-gems.html).
```
Comment (sonalkr132) [inline security.md:133]: How about https://web.archive.org/web/20131125012205/http://www.rubygems-openpgp-ca.org/blog/nobody-cares-about-signed-gems.html

File: security.md [hunk 2]
```diff
@@ -185,9 +185,7 @@ Credits
 Several sources were used for content for this guide:
 
 * [How to cryptographically sign your RubyGem](http://www.benjaminfleischer.com/2013/11/08/how-to-sign-your-rubygem-cert/) - Step-by-step guide
-* [Signing rubygems - Pasteable instructions](https://developer.zendesk.com/blog/2013/02/03/signing-gems/)
```
Comment (sonalkr132) [inline security.md:188]: https://web.archive.org/web/20130218074304/https://developer.zendesk.com/blog/2013/02/03/signing-gems/

File: security.md (hunk 1 above)
Comment (spk) [reply security.md:133]: Good idea i've amended the commit in 1c4eee3 and added `(archive)` as info on the link

**Comment by sonalkr132** [comment]
Thank you.

**Pull request merged**